src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -35,9 +35,22 @@ import { saveProfilePicture } from 'src/utils/image'
 import Logger from 'src/utils/Logger'
 import { useAsyncKomenciReadiness } from 'src/verify/hooks'
 import { Statsig } from 'statsig-react-native'
+import { ADJECTIVES, NOUNS } from './constants'
 
 type Props = StackScreenProps<StackParamList, Screens.NameAndPicture>
 
+//Exported for testing purposes only
+export const generateUsername = (): string => {
+  const adjective = ADJECTIVES[Math.floor(Math.random() * ADJECTIVES.length)]
+  const noun = NOUNS[Math.floor(Math.random() * NOUNS.length)]
+  const username = `${adjective} ${noun}`
+  const forbiddenUsernames: string[] = [] //TODO: Obtain username blacklist from Firebase Remote Config
```

Comment:
if you used a function to get the forbidden usernames (more realistic anyway), you could mock it and write a test that covers it, so we're all set for that later on once we fill in how that function will work

Reply:
alternatively, you can give the forbidden usernames as a parameter, which also lets us use it for testing. this is probably better, since i'm guessing we will want to get them with a hook anyway